### PR TITLE
replace validates_presence_of syntax

### PR DESCRIPTION
**Pull request by lokeshh**
Fixes https://www.pivotaltracker.com/story/show/146610007

(but see also https://www.pivotaltracker.com/story/show/145431073)

### CODE REVIEW

**Comment by lokeshh** [comment]
I'm not sure if this is what was intended. I'm confused why the pull request was referred in the issue tracker. Do I need to make the changes to the Pull Request? That seemed unlikely, so I made changed to the existing code where `validates_present_of` was used.

**Comment by tansaku** [comment]
I think this refactoring came up in the context of the events feature which hasn't been merged yet - that's our bad - but this is a great fix by itself @lokeshh - if you can join a scrum we can discuss in more detail.

Also note that `[Finishes #145431073]` should be in the commit message, but don't worry it's really fantastic to see your first PR! :-)

**Comment by lokeshh** [comment]
@tansaku I've updated the commit message. Thank you!

File: app/models/page.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class Page < ActiveRecord::Base
   #attr_accessible :content, :name, :permalink, :link_visible
-  validates_presence_of :name, :permalink
+  validates :name, presence: true
+  validates :permalink, presence: true
   validates_uniqueness_of :permalink
```
Comment (tansaku) [inline app/models/page.rb]: for another pull request maybe, but I think we could now replace:

```
validates_uniqueness_of :permalink`
```
with

```
validates :permalink, presence: true, unique: true
```

**Comment by tansaku** [comment]
great work @lokeshh - I think there are some other related refactorings if you are interested, but I'll get this merged in :-)

**Pull request merged**